JavaTaskSwitcher.java

```diff
@@ -0,0 +1,83 @@
+/***
+ * Today we are doing five tasks. Each task perform different function .
+ * All Task done without using for/while loops in the function
+
+ * Program owner- Kalpana Swami
+ * Date : 11 september 2024
+ */
+
+import java.util.*;
+public class JavaTaskSwitcher {
+    private static int expand(String s, int i, int j) {

```

Comment:
these is not good variable names, I should be able to understand the variable's use from just the middle of the code
Also the method name could be more descriptive like expandPalindrome